### ISSUE TITLE
Replacing the retired LUIS

### ISSUE THREAD

**Issue by ilmartin** [microsoft/BotFramework-Composer#9576]
Dear BotFramework-Composer Team,

I have been using BotFramework Composer to create Bot and deploy them to Azure, which needs the LUIS-Understanding resource created in Azure. However, I noticed that it is no longer available to create the LUIS resource today, which I suppose it means that it is not possible to fill in the LUIS resource credential in the configuration of the Bot unless I have an existing LUIS resource created in Azure to reference to. The following is the message in Azure:

> Language Understanding Intelligent Service (LUIS) is being retired on 1st October, 2025. A newer version of this capability is now available as a part of Azure Cognitive Service for Language called Conversational Language Understanding (CLU). To use this service, you need to provision a Language resource. You can't create new LUIS resources anymore. For information on migrating your existing LUIS apps to CLU projects consult the migration guide.

I would like to ask if this is the case or if there is an alternative way to continue to use LUIS. If not, when will the BotFramework-Composer be able to use the new CLU resource?

**Comment by anishprasad01** [comment]
Hi @ilmartin,

You can make use of CLU in Composer using the new [CLU Recognizer](https://github.com/microsoft/botframework-components/tree/main/packages/Recognizers/ConversationLanguageUnderstanding/dotnet).

Keep in mind that this recognizer is in the early stages, and not every feature may be 100% working yet.

**Comment by ilmartin** [comment]
Hi @anishprasad01 ,

Thank you for the solution, and sorry for the delay in response.

I have tried the following the GitHub readme file, I have successfully installed the package in the project and added the configuration to the recognizer. But when I try sending a message to the bot on Azure, it returns `Response status code does not indicate success: 404 (Not Found)`. I have definitely entered the correct CLU endpoints and keys, I believe it is a reference to the correct endpoints.

Secondly, I realise when I create a new Trigger with intent recognized, it no longer lets me enter Trigger phrases.
